tools/cuds-prioritized-attribution/test/tests/test_11_1/export.json

```diff
@@ -0,0 +1,322 @@
+{
+    "billing_account_id": "00F3D8-28D652-3F036F",
```

Comment:
Would it break tests to redact billing account in all these json?

Reply:
@jaketf What is the concern. These are dummy data.

Reply:
My concern is that this was a real billing account id.
If this is bogus then it's fine to leave it.